src/styles/index.ts

```diff
@@ -1746,6 +1749,31 @@ const styles = (theme: ThemeColors) =>
             lineHeight: variables.fontSizeOnlyEmojisHeight,
         },
 
+        emojisWithTextFontSizeAligned: {
+            fontSize: variables.fontSizeEmojisWithinText,
+            marginVertical: -7,
```

Comment:
@VickyStash @parasharrajat what was this vertical margin fixing? coming from https://github.com/Expensify/App/issues/53451

Reply:
thank you!